### PR TITLE
1 Add Fr Localisation

### PR DESCRIPTION
**Pull request by Fr-Dae**
https://github.com/Fr-Dae/factorio-colorblind-ultimate-dae/issues/1
https://github.com/Rycieos/factorio-colorblind-ultimate/issues/7

2nd attempt 
adding Fr localisation

### CODE REVIEW

**Comment by Fr-Dae** [comment]
no idea why i have conflict here, help needed.
thanks you for your kindess to tell me how make a nice PR =) with a dedicated branch it's much better.

File: data/core/settings.lua [hunk 1]
```diff
@@ -77,7 +77,6 @@ add_core_color_setting(
   "enemy_territory",
   { "", { "gui-map-generator.enemy-expansion-group-tile" }, " ", { "description.range" } }
 )
---add_core_color_setting("pollution", { "item-name.pollution" })
```
Comment (Fr-Dae) [inline data/core/settings.lua:80]: strange, i don't have touch to this file

**Comment by Fr-Dae** [comment]
i have cancel modification of dates format, and change 4 space for 2, like your preferencies.
tell me if you see other think i should fix

**Comment by Rycieos** [comment]
> no idea why i have conflict here, help needed. thanks you for your kindess to tell me how make a nice PR =) with a dedicated branch it's much better.

No worries, I will resolve the conflict by merging on my end. I'll get your translation in as part of v2.2.0. See commit 8f5aa7bf215ccfcac09b604584c76ad6ff037d85.

**Pull request closed without merging**